src/EA.Iws.Web/Areas/NotificationMovements/Views/PrenotificationBulkUpload/Index.cshtml

```diff
@@ -6,6 +6,16 @@
     Layout = "~/Views/Shared/_Layout.GovUK.Sections.cshtml";
 }
 
+<style>
+
+    .start-button
+    {
+        padding-left: 60px !important;
+        padding-right: 60px !important;
+    }
+
+</style>
+
 <div class="grid-row">
     <div class="column-two-thirds">
```

Comment:
I think the button's been resized due to the two-thirds style... Can we just get rid of this? Sorry I know I made the initial change for this !

Reply:
@GonzaloGirasol - replaced the two-thirds css class reference with full, but this didn't resolve the button sizing. Sizing is based on button text and as the text is just the word "Start" this is the reason for the default sizing being too small.